messaging/src/main/java/org/axonframework/queryhandling/SimpleQueryUpdateEmitter.java

```diff
@@ -105,7 +118,7 @@ public boolean queryUpdateHandlerRegistered(SubscriptionQueryMessage<?, ?, ?> qu
         };
 
         return new UpdateHandlerRegistration<>(registration,
-                                               processor.replay(updateBufferSize).autoConnect());
```

Comment:
Guessing that after setting the `axon_code_style.xml`, these changes would be reverted.